recipes/khiopsdriver-gcs/conda_build_config.yaml

```diff
@@ -0,0 +1,13 @@
+---
+cxx_compiler:
+  - gxx # [linux]
+  - clangxx # [osx]
+  - vs2022 # [win]
+cxx_compiler_version:
+  - 12 # [linux]
+  - 16 # [osx]
+
+c_stdlib_version:          # [osx and x86_64]
+  - "10.15"                # [osx and x86_64]
```

Comment:
```suggestion
```
Please remove these lines and adjust the `CXXFLAGSS` as I suggested in `build.sh`.